contracts/rollupReddit.sol

```diff
@@ -157,13 +160,14 @@ contract RollupReddit {
             bool
         )
     {
-        Types.Transaction memory _tx = RollupUtils.TxFromBytes(txBytes);
-        _tx.signature = sig;
+        bytes memory txs = RollupUtils.CompressTransferFromEncoded(txBytes);
+        // Validate ECDSA sig
```

Comment:
This is the offchain `processTransferTx`. It uses a `CompressTransferFromEncoded` here. Let me know what do you think @vaibhavchellani @kilic ?